stl/inc/algorithm

```diff
@@ -6927,9 +6927,7 @@ namespace ranges {
             const auto _Count = _RANGES distance(_Range);
             auto _UResult =
                 _RANGES _Lower_bound_unchecked(_Ubegin(_Range), _Count, _Val, _Pass_fn(_Pred), _Pass_fn(_Proj));
-            auto _Result = _RANGES begin(_Range);
-            _Seek_wrapped(_Result, _STD move(_UResult));
-            return _Result;
+            return _Rewrap_iterator(_Range, _STD move(_UResult));

```

Comment:
```suggestion
            return _RANGES _Rewrap_iterator(_Range, _STD move(_UResult));
```

same below

Reply:
I'm uncomfortable with how we've been spot-qualifying `_Ugly` functions - it would seem better to convert whole files at a time - but I can change all of our calls to `_Rewrap_iterator` across 3 files.

Reply:
We talked about this on Discord and agreed to perform mechanical qualifications in a separate PR to avoid mixing that with logic changes, even simple ones.